app/lib/frontend/templates/_utils.dart

```diff
@@ -12,11 +12,18 @@ import '../../shared/markdown.dart';
 const HtmlEscape htmlAttrEscape = HtmlEscape(HtmlEscapeMode.attribute);
 
 /// Renders a file content (e.g. markdown, dart source file) into HTML.
-String renderFile(FileObject file, String baseUrl) {
+String renderFile(FileObject file, String baseUrl, {String packageName}) {
   final filename = file.filename;
-  final content = file.text;
+  String content = file.text;
   if (content != null) {
     if (_isMarkdownFile(filename)) {
+      // Remove leading package name from the beginning of the markdown file.
+      if (packageName != null) {
+        final regexp = RegExp('#+ $packageName\n');
```

Comment:
It feels more right to parse the markdown... but if we really wanted to do this, then we should at-least use:
https://api.dart.dev/stable/2.7.1/dart-core/RegExp/escape.html

even if package names can only be safe strings...